src/qt/optionsmodel.cpp

```diff
@@ -243,6 +171,7 @@ static const QString GetDefaultProxyAddress()
 // read QSettings values and return them
 QVariant OptionsModel::data(const QModelIndex & index, int role) const
 {
+    int64_t prune = -1;
```

Comment:
The scope of `prune` can be limited to the scope of `case Prune` and `case PruneSize` below :-)